gui-tk/minsky.tcl

```diff
@@ -703,6 +703,12 @@ proc cut {} {
     minsky.cut
 }
 
+# Paste item(s) at position x and y on the canvas upon click down. For ticket 1098
+proc pasteAt {} {
+	canvas.mouseDown [get_pointer_x .wiring.canvas] [get_pointer_y .wiring.canvas]
+	minsky.paste
```

Comment:
Missing the new coordinate arguments

Reply:
It seems to me you still want pasteAt for ^V handling.

Reply:
Why is that there? I don't understand. They don't show up in my local bugfix-1098 branch?? I have also added the bugfix-901 changes because you have just merged that PR into your master branch. maybe I should revert and git pull your master into my local bugfix-1098 again? It has become a bit messy...